examples/helia-101/103-glob-unixfs.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-console */
+// @ts-check
+
+import * as nodefs from 'fs'
+import { pipeline } from 'stream/promises'
+import { createHeliaHTTP } from '@helia/http'
+import { car as CAR } from '@helia/car'
+import { unixfs, globSource } from '@helia/unixfs'
+
+// This is a script showing how to use globSource to merkelize files and directories from your local file system into 
+// Take the first argument as the path of the folder to add
+const args = process.argv.slice(2)
+
+// Example of how to use arguments
+if (args.length === 0) {
+  console.error('No argument provided')
+  console.error('Usage: node 103-glob-unixfs.js <path-to-add> <output-car-file>')
+}
+
+const path = args[0]
+const outputCarFile = args[1]
+
+// Check if the path exists
+if (!nodefs.existsSync(path)) {
+  console.error('Path does not exist')
+  process.exit(1)
+}
+
+const helia = await createHeliaHTTP()
+
+// UnixFS allows you to encode files and directories such that they are addressed by CIDs and can be retrieved by other nodes on the network
```

Comment:
Not a hard/fast rule but it's nice to limit comment line length to 80 chars for readability.  You can configure vscode to put a ruler in that helps.